docs/target-compatibility.md

```diff
@@ -56,6 +57,14 @@ tex.GetDimensions is the GetDimensions method on 'texture' objects. This is not
 
 SM6.5 Wave Intrinsics are supported, but requires a downstream DXC compiler that supports SM6.5. As it stands the DXC shipping with windows does not. 
 
+## WaveShuffle
+
+WaveShuffle is an intrinsic added to the Slang stdlibrary to expose the glsl `subgroupShuffle` intrinsics and allow loosened requirements on laneId. 
+
+In `HLSL` requires the `laneId` must be 'dynamically uniform' across the wave. WaveShuffle has the same functionality but relaxes this restriction. 
```

Comment:
Unrelated to this change, but we'll probably eventually need to normalize on one dialect of English when it comes to single-vs-double quotation marks, etc. The British conventions are certainly more logical, and I can't really complain when you are writing so much of our documentation, so I'm not inclined to try to impose anything too soon.

If/when we have resources to spare it would be good to assign somebody to scrub all our documents and get them into an explicitly chosen style.